### DISCUSSION TITLE
Send batch messages to kafka producer not working

### DISCUSSION THREAD

**Discussion by divyanshurai07** [apache/camel-karavan#739] [Q&A]
Hi , i am using kafka component for sending batch messages to kafka producer, but i am not able to send even after configuring it according to the documentation - https://camel.apache.org/components/3.20.x/kafka-component.html#_configuring_component_options

please suggest how the component is to be used in Karavan to ensure batch processing.

**Comment by davsclaus** [comment]
You can get help on general Camel here
https://camel.zulipchat.com/
